scripts/core/context.yml

```diff
@@ -228,12 +228,15 @@ params:
       name: hNativeContext
       desc: |
             [in][nocheck] the native handle of the context.
+    - type: $x_platform_handle_t
+      name: hPlatform
```

Comment:
making this an adapter instead, it makes the sycl changes cleaner and only needs a slight modification to the conformance test fixtures